galaxy/Dockerfile

```diff
@@ -19,7 +19,7 @@ GALAXY_CONFIG_DIR=/etc/galaxy
 ENV GALAXY_CONFIG_FILE=$GALAXY_CONFIG_DIR/galaxy.ini \
 GALAXY_CONFIG_JOB_CONFIG_FILE=$GALAXY_CONFIG_DIR/job_conf.xml \
 GALAXY_CONFIG_JOB_METRICS_CONFIG_FILE=$GALAXY_CONFIG_DIR/job_metrics_conf.xml \
-GALAXY_VIRTUALENV=$GALAXY_ROOT/.venv/ \
+GALAXY_VIRTUAL_ENV=/galaxy_venv \
```

Comment:
FYI, here are some practical notes regarding getting the `.venv` setup outside of Docker. These are based on the updates to the original PR from @jmchilton:

 * `/galaxy-central/scripts` needs to be at least temporarily copied out of the container. Otherwise, one can't run the shell scripts mentioned [here](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/admin/framework_dependencies.rst#managing-dependencies-manually).
 * `lib/galaxy/dependencies/pinned-requirements.txt` needs to be copied to `/galaxy-central/requirements.txt` or the equivalent location, since `lib/` doesn't exist outside of docker.
 * `common_startup.sh` ends up calling a python script that errors with:

```
Traceback (most recent call last):
    File "<string>", line 1, in <module>
ImportError: No module named galaxy.dependencies
```
 The part of the script that calls this is:

```
if [ $FETCH_WHEELS -eq 1 ]; then
    pip install -r requirements.txt --index-url ${GALAXY_WHEELS_INDEX_URL}
    GALAXY_CONDITIONAL_DEPENDENCIES=`PYTHONPATH=lib python -c "import galaxy.dependencies; print '\n'.join(galaxy.dependencies.optional('$GALAXY_CONFIG_FILE'))"`
    [ -z "$GALAXY_CONDITIONAL_DEPENDENCIES" ] || echo "$GALAXY_CONDITIONAL_DEPENDENCIES" | pip install -r /dev/stdin --index-url ${GALAXY_WHEELS_INDEX_URL}
fi
````
This can likely be manually run by doing the following inside docker and then installing the results listed outside:

```
source /galaxy-central/.venv/bin/activate
PYTHONPATH=lib python -c "import galaxy.dependencies; print '\n'.join(galaxy.dependencies.optional('$GALAXY_CONFIG_FILE'))"
```
For me, this was a simple `pip install --index-url https://wheels.galaxyproject.org/simple psycopg2==2.6.1 drmaa`